.github/workflows/refapp-3x-patient-registration.yml

```
@@ -1,9 +1,9 @@
 name: RefApp 3.x Patient Registration
 on:
   push:
-    branches: [ master ]
+    branches: [ dockerize-3.x ]
```

Keep master too.
```suggestion
    branches:
       - master
       - dockerize-3.x
```
or,
```suggestion
    branches: [ master, dockerize-3.x ]
```

---

applied to the pull request section as well